File: 学校上课写的/7.计导的斐波那契/fibonacci_1.go
package main

import (
	"fmt"
	"time"
)

func fibonacci(term int) int {
	if term <= 1 {
		return 1
	} else {
		return fibonacci(term-1) + fibonacci(term-2)
	}
}

func main() {
	var n int
	fmt.Printf("请问你想看到斐波那契数列的第几项: ")
	fmt.Scanf("%d", &n)
	start := time.Now()
	for i := 1; i <= n; i++ {
		fmt.Printf("F[%d] = %d\n", i, fibonacci(i))
	}
	end := time.Now()
	fmt.Printf("耗时: %dms\n", end.Sub(start).Milliseconds())
}

File: 学校上课写的/7.计导的斐波那契/Fibonacci_1.c
#include <stdio.h>
#include <time.h>

long long fibonacci(int n){
    if(n <= 2)
        return 1;
    else
        return fibonacci(n - 1) + fibonacci(n - 2);
}

int main(){
    int term, end, start;
    printf("你想看到斐波那契数列的第几项:");
    scanf("%d", &term);
    start = clock();
    for(int i = 1; i <= term;i++)
        printf("F[%d] = %lld\n", i, fibonacci(i));
    end = clock();
    printf("耗时: %dms\n", end - start);
}